.docker/setup_config.sh

```diff
@@ -19,6 +19,8 @@ conda install -c conda-forge cryptography=41.0.4 wheel=0.40.0
 rm -rf /root/miniconda-23.1.0/pkgs/cryptography-38.0.4-py39h9ce1e76_0
 rm -rf /root/miniconda-23.1.0/pkgs/wheel-0.37.1-pyhd3eb1b0_0
 rm -rf /root/miniconda-23.5.2/pkgs/cryptography-39.0.1-py39h9ce1e76_2
+rm -rf /root/miniconda-23.5.2/pkgs/certifi-2023.5.7-py39h06a4308_0
+rm -rf /root/miniconda-23.5.2/pkgs/conda-23.5.2-py39h06a4308_0/lib/python3.9/site-packages/tests/conda_env/support/requirements.txt
```

Comment:
Remove all tests! Why do we need to keep the other files in `support`?
This is production, we are not developing any code, let alone conda code.
We don't need any tests

@MukuFlash03 for visibility.

```suggestion
rm -rf /root/miniconda-23.5.2/pkgs/conda-23.5.2-py39h06a4308_0/lib/python3.9/site-packages/tests/
```